migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc

```diff
@@ -1,8 +1,36 @@
 :_mod-docs-content-type: ASSEMBLY
 [id=ossm-migrating-gateways-assembly]
-= Migrating gateways to Service Mesh 3.0
+= Migrating gateways from Service Mesh 2 to Service Mesh 3
 include::_attributes/common-attributes.adoc[]
 :context: ossm-migrating-gateways-assembly
 
 toc::[]
 
+This section is if you have been using gateways with {SMProductName} {SMv2Version} and are migrating them to {smproduct} 3.0. Migrating gateways between {SMProduct} 2 and {SMProduct} 3 is very similar to migrating regular workloads.
+
+There are three scenarios for migrating gateways from {SMProductName} 2 to {SMProduct} 3:
+
+* Gateway canary migration
+* Dedicated application gateway migration
+* Shared gateway migration
```

Comment:
There's really just two procedures: 1. canary and 2. in place. The difference between dedicated and shared here is just in how the gateway is deployed and how many applications are running traffic through it but the migration procedure is the same for both. You can get rid of one of these two sections and maybe rename the remaining one "in place" to contrast with the above "Gateway canary migration".